### ISSUE TITLE
Investigate Digest Word files provided by Stuart

### ISSUE THREAD

**Issue by Melissa37** [elifesciences/digest-parser#1]

**Comment by gnott** [comment]
I tried a few first methods of parsing the ``.docx`` file content. The best and simplest result I think is using the https://pypi.python.org/pypi/python-docx library.

Tried https://pypi.python.org/pypi/PyDocX and its ability to convert docx to HTML. The output looked good, but the tagging was not exact (multiple ``<em>`` and ``<strong>`` tags around content) which would probably take more time to clean up and convert to the final output format.

Tried parsing the XML content from the docx zip file using BeautifulSoup directly. This did produce a document, but the output is lots of tags and a mess which the ``python-docx`` is already interpreting into something more useful.

The result of coding a little further I have a quick prototype that wraps bold and italic styles around the appropriate content (omitting additional open and close tags when possible).

It should be easy to continue this to understand the specific names of sections and the specific terms inside them to extract the individual data fields.

For the purposes of investigating I think this issue is completed!
